operator/konflux.bundle.Dockerfile

```diff
@@ -20,6 +20,8 @@ ARG OPERATOR_IMAGE_TAG
 ARG OPERATOR_IMAGE_DIGEST
 ARG OPERATOR_IMAGE_REPO
 
+ARG RELATED_IMAGE_MANAGER
+ENV RELATED_IMAGE_MANAGER=$RELATED_IMAGE_MANAGER
```

Comment:
No. There's no RELATED_IMAGE_MANAGER or anything like that for the operator itself in `spec.install.spec.deployments[0].spec.template.spec.containers[0].env`.

If you think about it, it does not make sense. RELATED_IMAGE_* env variables are recognized by the operator and used by it as overrides to deploying operands. Operator is not operand and so listing it as RELATED_IMAGE_* results in no-op technically but confusion for future maintainers.

Once again referring you to look at the actual CSV from the downstream release.

I understand that declaring `RELATED_IMAGE_MANAGER` for the operator is a way to get it to `spec.relatedImages`, but there could be other ways to achieve that.

Reply:
Are you saying the variable should simply have a different name?

Reply:
A bit confusing indeed, that this is for setting up `spec.relatedImages.manager`, but it shouldn't be `RELATED_IMAGE_MANAGER`?

Reply:
>  but it shouldn't be RELATED_IMAGE_MANAGER?

That's what I believe in and explained above. Rather than taking my word on it, I suggest you check downstream CSV.

Reply:
Downstream shows the functioning example to follow. The semantics are to be learned from researching documentation and code. Even at the early stage of understanding when the semantics aren't known, the example serves as a substantial guidance.

Reply:
Again, you said *CSV*. I am just trying to untie some knots here and unless I am overlooking something critical, then the suggestion to look at the "downstream CSV" would be not helpful when it comes to researching about use-cases of these environment variables outside of the scope of the patching CSVs.

Reply:
Not sure I can follow the reasoning. CSV is the only authoritative place that sets RELATED_IMAGE_* environment variables. One can see which values are set and what they are set to.

How the variables are used is not defined by CSV but the fact that RELATED_IMAGE_ for the operator is absent in the list is apparent just from that.

Was my suggestion to use downstream CSV as the reference example misleading? I'm not convinced that it was.

Reply:
> How the variables are used is not defined by CSV [...]

That is exactly the point I was trying to make.